tests/data/README.md

```diff
@@ -206,6 +206,7 @@ This is allowed per the specification. libavif should decode such files
 correctly (i.e.) it should report them as files with alpha channel.
 
 Box structure of the items in this file:
+
```

Comment:
This seems unrelated, was it meant for this change?

Reply:
Yes, this is an unrelated change and will be merged as a separate commit. I added this commit to this pull request to force the AppVeyor build to run again. This fixes the rendering of this README.md file inside Google.

Please review this commit as an independent change. Thanks.

Reply:
Ack, please merge this PR without squashing the commits.